swap_meet/vendor.py

```diff
@@ -1,2 +1,73 @@
+# from .
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory=None):
+        inventory = [] if inventory is None else inventory
+        self.inventory = inventory
+    
+    def add(self, new_item):
+        self.inventory.append(new_item)
+        return new_item
+    
+    def remove(self, new_item):
+        if new_item in self.inventory:
+            self.inventory.remove(new_item)
+            return new_item
+        return False
```

Comment:
I'd consider flipping the logic so that your 'main' logic on lines 13-14 are not inside a conditional if block, but in the main level of indentation. 

Then you'd be able to use a guard clause to check if an item is not in inventory and immediately return False 

```suggestion
        if new_item not in self.inventory:
            return False
        self.inventory.remove(new_item)
        return new_item
```